app/services/force/lease_service.rb

```diff
@@ -30,28 +31,27 @@ def submit_lease(lease, application_id, primary_contact_id)
     private
 
     def create_lease(lease, application_id, primary_contact_id)
-      @client.create!('Lease__c',
-                      Application__c: application_id,
-                      Tenant__c: primary_contact_id,
-                      Unit__c: lease[:unit],
-                      Lease_Status__c: 'Draft',
-                      Lease_Start_Date__c: lease[:leaseStartDate],
-                      Monthly_Parking_Rent__c: lease[:monthlyParkingRent],
-                      Total_Monthly_Rent_without_Parking__c: lease[:totalMonthlyRentWithoutParking],
-                      Monthly_Tenant_Contribution__c: lease[:monthlyTenantContribution])
+      create_lease_params = lease_base_params(lease, application_id).merge(Tenant__c: primary_contact_id, Lease_Status__c: 'Draft')
```

Comment:
A little refactor here and on update so we can clearly see the difference in the params.